common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -775,6 +775,18 @@ def index_dictionary(self):
 
         return xblock_body
 
+    def set_discussion_status(self, is_enable_discussion):
+        ''' Enable or disable the disussion.
+        '''
+        for child in self.get_children():
+            child.set_discussion_status(is_enable_discussion)
+
+    def get_discussion_status(self):
+        for child in self.get_children():
+            if not child.get_discussion_status():
+                return False
+        return True
```

Comment:
We discussed this. This is not a True/False thing. It should also return if _some_ of the units have it enabled.